README.rst

```diff
@@ -42,6 +42,22 @@ From the command-line, this requires some configuration::
     key = cloudstack api key
     secret = cloudstack api secret
 
+The config can also specify no ssl verification:
+    cat $HOME/.cloudstack.ini
+    [cloudstack]
+    endpoint = https://api.exoscale.ch/compute
+    key = cloudstack api key
+    secret = cloudstack api secret
+    verify = False
```

Comment:
I'm all for being able to specify your own CA and possibly client cert, but I'd like to avoid making it so easy to pass `verify=False`. Basically, default to `verify=True` and allow passing `verify=/path/to/ca.pem` but not `verify=False`.

Additionally, being able to pass `cert = /path/to/client-cert-bundle.pem` would be nice.